bindata/recovery/upgrade-recovery.sh

```diff
@@ -175,15 +175,11 @@ function take_backup {
     fi
 
     echo "##### $(date -u): Backing up container images, cluster, and required files"
-    # Note: The cluster-restore script uses a hardcoded path for manifests-stopped dir
-    rm -rf ${BACKUP_DIR} /home/core/assets/manifests-stopped
 
-    mkdir -p ${BACKUP_DIR}
-
-    time for id in $(crictl images -o json | jq -r '.images[].id'); do
-        mkdir -p ${BACKUP_DIR}/containers/$id
-        /usr/bin/skopeo copy --all --insecure-policy containers-storage:$id dir:${BACKUP_DIR}/containers/$id
-    done
+    #time for id in $(crictl images -o json | jq -r '.images[].id'); do
```

Comment:
Overall, I'd rather remove this rather than leave this commented as this could grow large with garbage code quite easily.

Reply:
I have uncommented the image backup/restore blocks and made it an option when running the script